src/EnergyPlus/AirflowNetwork/src/Solver.cpp

```diff
@@ -6821,10 +6821,10 @@ namespace AirflowNetwork {
 
         state.dataCurveManager->PerfCurve(CurveNum).InterpolationType = CurveManager::InterpType::BtwxtMethod;
 
-        state.dataCurveManager->PerfCurve(CurveNum).Var1Min = 0.0;
-        state.dataCurveManager->PerfCurve(CurveNum).Var1MinPresent = true;
-        state.dataCurveManager->PerfCurve(CurveNum).Var1Max = 360.0;
-        state.dataCurveManager->PerfCurve(CurveNum).Var1MaxPresent = true;
+        state.dataCurveManager->PerfCurve(CurveNum).inputLimits[0].min = 0.0;
```

Comment:
I can already tell this PR is going to be 🔥

Reply:
No `auto &thisCurve`?

Reply:
In general, outside of CurveManager.cc and CurveManager.hh, I tried to touch the code as little as possible.  Of course this would be a fine addition, but unless I have to muck around in there again for this PR I'll just leave it.